### ISSUE TITLE
Typescript signatures don't allow instanceof checking for Error

### ISSUE THREAD

**Issue by akdor1154** [vitaly-t/pg-promise#291]
The Typescript signatures cause a tsc error when trying to type-check a thrown error:
```ts
	import PGP = require('pg-promise');
	return db.one(SQL`...`)
	.catch( (e) => {
		if (e instanceof PGP.errors.QueryResultError) {
			...
		}
		throw e;
	})
```
`//error TS2359: The right-hand side of an 'instanceof' expression must be of type 'any' or of a type assignable to the 'Function' interface type.`

**Comment by vitaly-t** [comment]
Looks like an issue in the `errors` declaration.

**Comment by NobleUplift** [comment]
I have the exact same issue. Is there a workaround?

**Comment by vitaly-t** [comment]
I have merged it, will release an update shortly.

**Comment by vitaly-t** [comment]
This has been fixed and released in [v5.8.2](https://github.com/vitaly-t/pg-promise/releases/tag/v.5.8.2).
